include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -532,10 +529,16 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
     #endif
     
     /// Remove lines rejected by PolyFit from D_A1_FittedPos and D_A1_FittedWLen
-    #ifdef __DEBUG_IDENTIFY__
-        for (int i = 0; i < rejected->size(); ++i)
-            cout << "identify: rejected[" << i << "] = " << (*rejected)[i] << endl;
-    #endif
+    _mask.addMaskPlane("REJECTED_LINES");
+    unsigned short maskVal = 1 << _mask.getMaskPlane("REJECTED_LINES");
+    for (int i = 0; i < rejected->size(); ++i){
+        #ifdef __DEBUG_IDENTIFY__
+            cout << "identify: rejected D_A1_FittedPos[" << (*rejected)[i] << "] = " << D_A1_FittedPos[(*rejected)[i]] << endl;
+        #endif
+        for (int p = (D_A1_FittedPos[(*rejected)[i]-2] < 0 ? 0 : D_A1_FittedPos[(*rejected)[i]-2]);
+                 p < (D_A1_FittedPos[(*rejected)[i]+2] >= _length ? _length-1 : D_A1_FittedPos[(*rejected)[i]+2]); ++p)
+            _mask(p, 0) |= maskVal;
```

Comment:
The commit messages says you're just changing "_mask in Spectrum from ndarray to afwImage::Mask", but this looks very much like you're actually setting bits in the mask.  This should probably be a separate commit; if it isn't, then at the least this change should be mentioned in the commit message.